src/widgets/reflow.rs

```diff
@@ -189,6 +207,20 @@ impl<'a, 'b> LineComposer<'a> for LineTruncator<'a, 'b> {
     }
 }
 
+fn trim_width(src: &str, mut wid: usize) -> &str {
```

Comment:
Even though the function name is quite explicit, could you add a small doc comment on what this function does ?

Reply:
Okay.